tests/devapps/XForms/XForms/AcquirePage.xaml.cs

```diff
@@ -157,7 +189,7 @@ private string GetSelectedUserId()
             return UserNotSelected.Equals(selectedUserId, StringComparison.OrdinalIgnoreCase) ? null : selectedUserId;
         }
 
-        private async Task OnAcquireSilentlyClickedAsync(object sender, EventArgs e)
+        private async void OnAcquireSilentlyClickedAsync(object sender, EventArgs e)
```

Comment:
This is ok but it will generate a warning. Consider disabling this warning inline.